stripe/src/main/java/com/stripe/android/view/CountryTextInputLayout.kt

```diff
@@ -29,7 +31,7 @@ import kotlin.properties.Delegates
  * [R.styleable.StripeCountryAutoCompleteTextInputLayout_countryItemLayout], note this layout must
  * be a [TextView].
  */
-internal class CountryTextInputLayout @JvmOverloads constructor(
+internal open class CountryTextInputLayout @JvmOverloads constructor(
```

Comment:
this is pretty ugly, but seems it's pretty hard to mimic configuration in test, so this is the least intrusive way I can come up with to test it

Reply:
You should be able to test this with FragmentScenario and recreate.

https://developer.android.com/guide/fragments/test#recreate

I don't think we need to make this `open`.

Reply:
hmm I actually tried that, please see my comments in CountryTextInputLayoutTest, calling `ActivityScenario#recreate` would only trigger a CountryTextInputLayout instance to be recreated and its `onSaveInstanceState` being called, but never triggers `onRestoreInstanceState` called, so after recreating the new instance always has US selected by default. 
I also tried to manually `moveToState` or `requestedOrientation` but with no avail :(

Reply:
this is no longer required after we added a dedicated restore method